From: Alan Shearer <sakison@gmail.com>
Date: Tue, 13 Oct 2026 09:23:00 -0500
Subject: bluetooth: fix spelling in SDP client

Fix spelling mistakes in the Bluetooth SDP client.

TRACE_BT messages:
- Error occured -> occurred
- Issueing -> Issuing
- Recieved -> Received

Variable name:
- arrtRange -> attrRange

These TRACE strings show up in syslog when Bluetooth tracing is on.
No functional change.

diff --git a/src/servers/bluetooth/SDPClient.cpp b/src/servers/bluetooth/SDPClient.cpp
--- a/src/servers/bluetooth/SDPClient.cpp
+++ b/src/servers/bluetooth/SDPClient.cpp
@@ -62,7 +62,7 @@
 	status = getsockopt(fClientSocket, SOL_SOCKET, SO_ERROR, &sockError, &errorLength);
 	if (status < 0)
-		TRACE_BT("SDP: Error occured\n");
+		TRACE_BT("SDP: Error occurred\n");
 	else if (sockError == 0)
 		TRACE_BT("SDP: Socket connected.\n");
 	else
@@ -105,9 +105,9 @@
 	rawReqDataIO.WriteExactly(&maxAttrByteCount, 2);
 
 	// getting all attributes start_id = 0x0000, end_id = 0xFFFF
-	uint32 arrtRange = 0x0000FFFF;
+	uint32 attrRange = 0x0000FFFF;
 	rawReq.AddDataSeq(5);
-	rawReq.AddUInt32(arrtRange);
+	rawReq.AddUInt32(attrRange);
 
 	char contStateData[17];
 	uint8 contStateNumBytes = 0;
@@ -140,7 +140,7 @@
 		BMallocIO reply;
 		if (_IssueRequest(request.Buffer(), request.BufferLength(), &reply) != B_OK) {
-			TRACE_BT("SDP: Issueing Request failed\n");
+			TRACE_BT("SDP: Issuing Request failed\n");
 			return B_ERROR;
 		}
 
@@ -248,7 +248,7 @@
 	if (numBytes < 0)
 		return B_ERROR;
 
-	TRACE_BT("SDP: Recieved the response: num bytes=%ld\n", numBytes);
+	TRACE_BT("SDP: Received the response: num bytes=%ld\n", numBytes);
 
 	reply->SetSize(0);
 	reply->Write(buffer, numBytes);
